apps/core/lib/core/snitch/domain/stock/quantifier.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Core.Snitch.Domain.Stock.Quantifier do
+  @moduledoc """
+    Interface for handling inventory related business logic
+  """
+  
+  use Core.Snitch.Domain
+
+  def total_on_hand(variant) when is_map(variant), do: total_on_hand(variant.id)
```

Comment:
1. This will only return the stock quantity total from _"active"_ stock locations. There is no method to obtain the total from _all_ locations. We could make this:
`def total_on_hand(variant_id, only_active \\ true)`
2. Please document these methods.